gio/Gir.toml

```diff
@@ -553,6 +554,15 @@ status = "generate"
     name = "no_match_rule"
     version = "2.70"
 
+[[object]]
+name = "Gio.DebugControllerDBus"
+status = "generate"
+manual_traits = ["DebugControllerDBusExtManual"]
+    [[object.property]]
+    name = "connection"
```

Comment:
hmm, not sure how it ended up marking the property as nullable :/ maybe it should be but then the constructor isn't so...

Reply:
I guess pointer properties are nullable by default unless overridden in the docs?

Reply:
Sadly using `(not nullable)` does not work on properties, it would be nice if it did